lessons/beginners/class/index.md

```diff
@@ -226,7 +226,7 @@ rozlišit.
 Teď se na chvíli vraťme k metodám. Konkrétně k parametru `self`.
 
 Každá metoda má přístup ke konkrétnímu objektu, na
-kterém pracuje, právě přes argument `self`.
+kterém pracuje, právě přes parametr `self`.
```

Comment:
Je někde vysvětlen rozdíl mezi parametery a argumenty? Tady (i v kapitole o funkcích) je to pochopitelně použito správně, ale nějak mi přijde, že se to nedefinuje (jsem ochoten u funkcí ty dvě tři věty doplnit, pokud ti to přijde jako fajn nápad).

Reply:
To myslím vysvětleno není. A já si je sám vždycky pletu...
Bylo by to fajn k definicím funkcí doplnit.